src/Microsoft.Identity.Client/ApiConfig/AcquireTokenSilentParameterBuilder.cs

```diff
@@ -56,6 +63,12 @@ private AcquireTokenSilentParameterBuilder WithLoginHint(string loginHint)
             return this;
         }
 
+        private AcquireTokenSilentParameterBuilder WithSingleAccount()
+        {
+            //CommonParameters.AddApiTelemetryFeature(ApiTelemetryFeature.WithLoginHint);
```

Comment:
I am assuming we want to add another telemetry entry here? @henrik-me @jmprieur

Reply:
makes sense yes.

---
In reply to: [288809223](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/1181#discussion_r288809223) [](ancestors = 288809223)